docs/root/start/simple_example.rst

```diff
@@ -0,0 +1,240 @@
+.. _start_simple:
+
+Simple Example
+==============
+
+This is perhaps the most minimal Envoy configuration which can be used to validate basic plain HTTP
+proxying. It is available in :repo:`configs/google_com_proxy.v2.yaml`. This is not
+intended to represent a realistic Envoy deployment.
+
+Quick Start
+-----------
+
+This Quick Start gets you started using only files in the Envoy repo. The sections below explain
+this configuration file and execution steps in detail.
+
+Copy both :repo:`configs/Dockerfile` and
+:repo:`configs/google_com_proxy.v2.yaml` to the same directory on your local
+disk. Then, build and run the Dockerfile, and test out Envoy by sending a
+request to port 10000::
+
+  $ docker build -t envoy-google-test:v1 .
+  $ docker run -d -p 10000:10000 envoy-google-test:v1
+  $ curl -v localhost:10000
+
+The Dockerfile will build a container containing the latest version of Envoy,
+and copy a basic Envoy configuration into the container. This basic
+configuration tells Envoy to route incoming requests to \*.google.com.
+
+
+Configuration Walkthrough
+-------------------------
+
+Envoy can be configured both statically and dynamically.
+In the static configuration model, a YAML configuration file is used, which is passed
+as an argument on the command line.
+Envoy also has an extensive set of APIs that can be used for dynamic configuration.
+In this section, we'll walk through a simple static configuration.
+
+Below is the `google_com_proxy.v2.yaml` file.
+
+.. code-block:: yaml
+
+  admin:
```

Comment:
I don't think the indent of this snippet is correct. Everything is nested under admin. Can you take a look? (In a real perfect world we would somehow load these snippets directly out of the files in the repo, but that sounds really complicated).

Reply:
Good catch (and embarrassing that I left it that way).